app/src/main/java/comp3350/mealbuddy/business/Calculator.java

```diff
@@ -266,4 +276,61 @@ private int getMealLabelCalories(Meal foodList, String label) {
         return totalCalories;
     }
 
+    /*
+     * getMets
+     * Get the MET value associated with an exercise intensity
+     * Parameters:
+     *     @param exercise - The exercise
+     * Return:
+     *     The met value for the exercise
+     */
+    private int getMets(Exercise exercise) {
+        int mets = -1;
+        switch(exercise.intensity) {
+            case Low:
+                mets = LOW_MET;
+                break;
+            case Medium:
+                mets = MEDIUM_MET;
+                break;
+            case High:
+                mets = HIGH_MET;
+                break;
+        }
+        return mets;
+    }
+    /*
+     * getExerciseCalories
+     * Get the calories burned in an exercise.
+     * Parameters:
+     *     @param exercise - The exercise
+     *     @param userInfo - The userInfo of the user performing the exercise
+     * Return:
+     *     The calories burned in the exercise
+     */
+    public int getExerciseCalories(Exercise exercise, UserInfo userInfo) {
+        int met = getMets(exercise);
+        return (int)(((userInfo.weight / KG_CONVERSION) * MET_MULTIPLIER * met * exercise.duration) / MET_DIVISOR);
+    }
+
+    /*
+     * getTotalExerciseCalories
+     * Get the total calories burned in all the exercises in a day.
+     * Parameters:
+     *     @param userInfo - The userInfo of the user performing the exercises
+     * Return:
+     *     The total calories burned in the exercises
+     */
+    public int getTotalExerciseCalories(UserInfo userInfo) {
+        int totalCalories = 0;
+        for (Exercise exercise : day.getExercises()) {
+            totalCalories += getExerciseCalories(exercise, userInfo);
+        }
+        return totalCalories;
+    }
+
```

Comment:
/*
     * getNetCalories
     * Get the net gain/loss of calories for a day
     * Parameters:
     *     @param userInfo - The userInfo of the user
     * Return:
     *     The net calories gained/lost for the day.
     */

Reply:
oops wrong place but im giving u a free commit to add this comment